compiler/test/__snapshots__/exceptions.a68ae348.0.snapshot

```diff
@@ -150,5 +150,5 @@ exceptions › exception_4
    (call $_gmain)
   )
  )
- ;; custom section \"cmi\", size 1238
+ ;; custom section \"cmi\", size 1284
```

Comment:
Why would these increase in size instead of decrease?

Reply:
We include location information in CMIs now (we previously stripped all location information), so sizes went up. This doesn't affect final binary sizes and will also help with LSP features, like go to definition.

Reply:
(I did this because the check just literates the signature, and I essentially am doing a "go to definition" to report the error wherever the error is.)